src/components/item/sharing/CategorySelection.js

```diff
@@ -148,6 +151,7 @@ function CategorySelection({ item, edit }) {
             <TextField
               {...params}
               variant="outlined"
+              id="selectedLevelValues"
```

Comment:
Don't hardcode ids!! Factor out in `selectors`.

Reply:
I actually find out this id not useful so I just deleted. Previously I added it just for testing ( to see if I could locate the component), and I forgot to remove it.